features/standby_cluster.feature

```diff
@@ -14,7 +14,7 @@ Feature: standby cluster
     Then "members/postgres0" key in DCS has state=running after 10 seconds
     And replication works from postgres1 to postgres0 after 15 seconds
 
-  @skip
```

Comment:
Maybe we better rename it to smth like `@skip-no-slot-advance`? In case we will need to skip another subset of test because of some other newly implemented features in the future

Reply:
The error msg 'Unsupported PostgreSQL version' is also seems to be misleading for me